src/http/config_http_service.cpp

```diff
@@ -37,4 +37,15 @@ void update_config(const http_request &req, http_response &resp)
     resp.body = out.str();
     resp.status_code = http_status_code::ok;
 }
+
+void list_all_configs(const http_request &req, http_response &resp)
+{
+    if (req.query_args.size() != 0) {
```

Comment:
Why check `query_args `?

Reply:
someone may misunderstand the usage of this interface, and maybe he will add a query arg to get a special config. the check is used for reminding him.